web/src/components/DatePicker.tsx

```diff
@@ -59,30 +58,38 @@ const StyledError = styled.div`
   font-weight: bold;
   color: ${props => props.theme.colors.invalidInput};
 `
+
 export type DatePickerProps = {
   title: string
   date?: DateTime | null
-  setDate: (startDate: DateTime | null) => void
+  setDate: (date: DateTime | null) => void
   error?: string
 }
+
 const DatePicker = ({ title, date, setDate, error }: DatePickerProps): ReactElement => {
+  const { t } = useTranslation('events')
+  const [tempDate, setTempDate] = useState(date?.toISODate() ?? '')
+  const isInvalidDate = tempDate !== '' && date?.toISODate() !== tempDate
```

Comment:
Sorry, I just realized that this leads to the error quickly flickering if you reset the filter. Using a `isValid` method fixes this issue:
```ts
const isValidIsoDate = (date: string): boolean => {
  try {
    DateTime.fromISO(date)
    return true
  } catch (_) {
    return false
  }
}
```
```suggestion
  const isInvalidDate = tempDate !== '' && isValidIsoDate(tempDate)
```